object.py

```diff
@@ -10,9 +10,6 @@
   lines_video = picture_lines(picture_video)
   region_place_part = region_place(lines, lines_video)
   draw_picture("/Users/nityaarora/Downloads/picture.png", region_place_part[0], region_place_part[1], region_place_part[2], region_place_part[3])
```

Comment:
Might want to use relative pathing instead of absolute pathing (replace `/Users/nityaarora/` with `~/` so that other people can run the code.